3ds Max/Max2Babylon/Exporter/MaxExportParameters.cs

```diff
@@ -1,9 +1,10 @@
-﻿using BabylonExport.Entities;
-
-namespace Max2Babylon
-{
-    public class MaxExportParameters : ExportParameters
-    {
-        public Autodesk.Max.IINode exportNode;
-    }
-}
+﻿using BabylonExport.Entities;
+
+namespace Max2Babylon
+{
+    public class MaxExportParameters : ExportParameters
+    {
+        public Autodesk.Max.IINode exportNode;
+        public bool useHoldFetchLogig = true; // some script could use the babaylon exporter and override this value
```

Comment:
is this a typo? "HoldFetchLogig"

Reply:
I misspelled, is useHoldFetchLogic
I added this parameter because I expect that many tools could use their Hold/Fetch system
In that case, passing through MaxScript Manager the internal hold/fetch of Babylon will be overridden

Reply:
I actually removed it, I  will do a second pr after to adapt the maxscript manager

Reply:
here some info about Hold/fetch system
https://knowledge.autodesk.com/support/3ds-max/learn-explore/caas/CloudHelp/cloudhelp/2016/ENU/3DSMax/files/GUID-4083A03C-E635-4911-9EAA-5791859EEC3E-htm.html